.cargo-task/generate-drivers/src/main.rs

```diff
@@ -0,0 +1,147 @@
+use std::{
+    env,
+    fs::{create_dir_all, remove_dir_all, File},
+    io::prelude::*,
+    path::PathBuf,
+};
+
+use serde_derive::Serialize;
+use serde_json::Value;
+use tinytemplate::{error::Error, format_unescaped, TinyTemplate};
+
+mod cargo_task_util;
+
+mod config;
+use config::{load_drivers_toml, Driver};
+
+fn main() -> std::io::Result<()> {
+    // `root`      - executing directory; assumed to be Step/Dir root
+    // `drivers`   - driver facade crate directory (output)
+    // `templates` - template directory (input)
+    let root = env::current_dir()?;
```

Comment:
Idea for future extension: Scan directory tree, looking for `drivers.toml`. Use that directory for `root`.